core/src/main/java/io/aiven/klaw/service/UsersTeamsControllerService.java

```diff
@@ -283,29 +283,25 @@ String generateRandomWord(int len) {
     return sb.toString();
   }
 
-  public ResetPasswordInfo resetPassword(String username) {
-    log.info("resetPassword {}", username);
+  //  public ResetPasswordInfo resetPassword(String userName,String token, String password) { }
+
+  public ResetPasswordInfo resetPasswordGenerateToken(String username) {
+    log.info("resetPasswordGenerateToken {}", username);
     ResetPasswordInfo resetPasswordInfo = new ResetPasswordInfo();
     UserInfoModelResponse userInfoModel = getUserInfoDetails(username);
-    resetPasswordInfo.setPasswordSent("false");
```

Comment:
can we delete this field passwordSent ?

Reply:
Password Sent has now been changed to TokenSent so the FE knows that the token was sent correctly